graphics/libmypaint/Makefile

```diff
@@ -11,9 +11,12 @@ COMMENT=	Brush library from the MyPaint project
 LICENSE=	ISCL
 LICENSE_FILE=	${WRKSRC}/COPYING
 
+BUILD_DEPENDS=	${LOCALBASE}/bin/automake:devel/automake \
+		python2:lang/python
```

Comment:
The convention is not to depend on python metaports, but to either patch the files to depend on the particular python or use the alias thing: https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/binary-alias.html#binary-alias-ex2

Reply:
OK done.